### PR TITLE
Bug 1069830 - Contact name form field's autocorrect feature makes entering foreign names difficult r=francisco

### PR DESCRIPTION
**Pull request by sergi**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4591be3ed8b0d51098ddf74b0ffc6029a7300df6

File: apps/communications/contacts/elements/form.html [hunk 1]
```diff
@@ -127,11 +127,11 @@ <h1 id='contact-form-title' data-l10n-id="addContact"></h1>
             </div>
             <div class="item-body-exp with-overflow">
               <p class="fillflow-row input-container">
-                <input placeholder="Name" data-field="givenName" data-l10n-id="givenName" name="givenName" class="textfield" type="text" x-inputmode="latin-prose" value="" id="givenName" dir="auto">
+                <input placeholder="Name" data-field="givenName" data-l10n-id="givenName" name="givenName" class="textfield" type="text" x-inputmode="latin-prose" value="" id="givenName" dir="auto" autocorrect="off" autocapitalize="off" spellcheck="false" x-inputmode="verbatim">
```
Comment (arcturus) [inline apps/communications/contacts/elements/form.html]: I think the autocapitalize off is not a good idea. Perhaps in some cases we want to have it in lower case, but 90% of the cases a name and surname will be capitalized

Comment (sergi) [inline apps/communications/contacts/elements/form.html]: Fair point, I'll enable it back.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8d2cbec17c274b5f785c6ff051f584a11aa1c9c8

**Pull request merged**